src/internal/components/structured-item/interfaces.ts

```diff
@@ -9,4 +9,5 @@ export interface StructuredItemProps {
   actions?: ReactNode;
   secondaryContent?: ReactNode;
   disablePaddings?: boolean;
+  disableActionsWrapping?: boolean;
```

Comment:
nit: should we call it `wrapActions` - similar to table's `wrapLines` (and also similar flex-wrap)? It can be true or false by default.